core/lib/libtomcrypt/src/math/fp/ltc_ecc_fp_mulmod.c

```diff
@@ -1436,8 +1436,10 @@ int ltc_ecc_fp_save_state(unsigned char **out, unsigned long *outlen)
     * and the mu INTEGER
     */
    cache_entry = XCALLOC(FP_ENTRIES*(2*(1U<<FP_LUT)+4)+3, sizeof(ltc_asn1_list));
-   if (cache_entry == NULL)
-      return CRYPT_MEM;
+   if (cache_entry == NULL){
```

Comment:
There's a space missing  between `)` and `{`

Reply:
OK